docs/source/workflow_inputs.rst

```diff
@@ -318,6 +318,10 @@ Electricity Rates
 
 For each scenario, electricity rates can be optionally entered as an ``/HPXML/SoftwareInfo/extension/UtilityBillScenarios/UtilityBillScenario/UtilityRate``.
 
+**Simple**
+
+If the utility rate type is simple, utility bills are calculated based on fixed charges and a marginal rate.
```

Comment:
We should probably move away from the simple vs detailed terminology. Yes, the tariff JSON files can be detailed, but they can be simple flat rates too. I will plan to address this.

Reply:
On second thought, it's pervasively used everywhere so I will just attempt to clarify what simple vs detailed means, as opposed to making lots of changes.